frontend/src/metabase/lib/colors/colors.ts

```diff
@@ -277,12 +277,12 @@ export const colorConfig = {
     dark: baseColors.lobster[90],
   },
   "background-hover": {
-    light: `color-mix(in srgb, var(--mb-color-brand) 15%, transparent)`, //baseColors.oceanAlpha[10],
-    dark: `color-mix(in srgb, var(--mb-color-brand) 15%, transparent)`, //baseColors.oceanAlpha[20],
+    light: `color-mix(in srgb, var(--mb-color-brand) 21%, transparent)`, //baseColors.oceanAlpha[10],
+    dark: `color-mix(in srgb, var(--mb-color-brand) 21%, transparent)`, //baseColors.oceanAlpha[20],
```

Comment:
These need to stay color-mixed so that we continue to leverage the `brand` color when whitelabelled, but we changed the alpha values to reflect that oceanAlpha uses